src/registry.js

```diff
@@ -132,6 +132,22 @@ exports.traceIs = function(traceType, category) {
     return !!_module.categories[category];
 };
 
+/**
+ * Retrieve component module method
+ *
+ * @param {string} name
+ *  name of component (as declared in component module)
+ * @param {string} method
+ *  name of component module method
+ * @return {function}
+ */
+exports.getComponentMethod = function(name, method) {
```

Comment:
Hmm. I don't know how I like this. Why not just return the component so `getComponent('component').method`? The string abstraction seems unnecessary.